benchmark-service/benchmark/benchmark.py

```diff
@@ -33,16 +92,43 @@ async def greet_user(request: web.Request) -> web.Response:
     return response
 
 
+@router.get('/name/{name}')
+@web_authenticated_developers_only(redirect=False)
+async def show_name(request: web.Request, userdata) -> web.Response:  # pylint: disable=unused-argument
```

Comment:
This is fine for now, but I think we'll need to pass a query parameter with the name of the file path to look at. Maybe that would be a good way to pass the benchmark name as well.

I think I like benchmark and benchmarks as the equivalent of a single benchmark test and benchmarks as a collection of benchmark tests.